lib/src/react_client/utils.dart

```diff
@@ -0,0 +1,234 @@
+@JS()
+library react_client_utils;
```

Comment:
In order to use the @JS() annotation for `_objectKeys` and `defineProperty`, the file had to be a library. Not sure if there's a better way to handle this?

Reply:
nope... this is fine.